tests/e2e/default-mobile/content-security-policy.wdio-spec.js

```diff
@@ -0,0 +1,54 @@
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+
+describe('Content Security Policy', () => {
+  const logEntries = [];
+  const getBrowserLogs = async (logLevels) => {
```

Comment:
This log code can be in `tests/utils/browser-logs.js` to reuse later.

Reply:
Thanks, @latin-panda! What do you think if I put this in `tests/utils/browser.js` or do you prefer a new file?

Reply:
Yes, that's still okay since it's about the browser's console log; it's intuitive when using it: `browser.getlogs(logLevels)`.

Looking more into `tests/utils/browser.js`, it has several functions related to CHT databases, `createDoc`, `updateDoc`, `getDoc`, etc. When you see the code `browser.updateDoc(myReport)`, it's not intuitive that it's working cht databases; people can think it's about javascript global `document`. 
In my opinion, it's better to extract these functions to a `/tests/utils/cht-db.js`, then people can use it as: `chtDB.updateDoc(myReport)` to increase understanding when reading the code <-- This can be a separate nice and tiny PR :)